### ISSUE TITLE
ssl (urllib3) InsecurePlatformWarning

### ISSUE THREAD

**Issue by kpcyrd** [Netflix/security_monkey#620]
I'm running the docker image and got this warning:
```
/usr/local/lib/python2.7/dist-packages/botocore-1.5.26-py2.7.egg/botocore/vendored/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
```

which, according to the linked page:
> InsecurePlatformWarning
>    This happens on Python 2 platforms that have an outdated ssl module. These older ssl modules can cause some insecure requests to succeed where they should fail and secure requests to fail where they should succeed. Follow the pyOpenSSL guide to resolve this warning.

That's probably worth fixing in the Dockerfile.

**Comment by mikegrima** [comment]
Hello @kpcyrd :

Can you submit a pull request to address this?

**Comment by monkeysecurity** [comment]
Non-docker is also seeing this now.

**Comment by monkeysecurity** [comment]
I believe I have fixed this in #667.  I would appreciate confirmation.
